packages/url_launcher/url_launcher/pubspec.yaml

```diff
@@ -26,7 +26,11 @@ dependencies:
   # validation, so we set a ^ constraint.
   # TODO(amirh): Revisit this (either update this part in the  design or the pub tool).
   # https://github.com/flutter/flutter/issues/46264
-  url_launcher_web: ^0.1.0+1
+  url_launcher_web: 
+    git: 
+        url: https://github.com/Vanethos/plugins
+        ref: feature/add-tel-url
+        path: packages/url_launcher/url_launcher_web
```

Comment:
This needs to be reverted, otherwise you'll never be able to publish the plugin.

Reply:
Note that `0.1.2` will be automatically picked up by url_launcher after it gets published. If you want to have this change locally for your own tests, it's fine, but you need to remove it from the PR, so the CI bots (publishable) don't complain at you.

Reply:
OH! SNAP! I edited it locally and did not commit it! will do it now